integration_spec/spec_helper.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+require_relative "supports/logstash_test_helpers"
+require "fileutils"
+
+# Configure the test environment
+source = File.expand_path(File.join(File.dirname(__FILE__), ".."))
+integration_path = File.join(source, "integration_run")
+
+puts "[integration_spec] configure environment"
+
+if Dir.exists?(integration_path)
+  # We copy the current logstash into a temporary directory
+  # since the tests are a bit destructive
+  FileUtils.mkdir_p(integration_path)
+  rsync_cmd = "rsync -a --exclude 'rspec' --exclude '#{File.basename(integration_path)}' --exclude 'integration_spec' --exclude '.git' #{source} #{integration_path}"
+
+  puts "[integration_spec] Rsync source code into: #{integration_path}"
+  system(rsync_cmd)
+  puts "[integration_spec] Finish rsync"
+
+  LOGSTASH_TEST_PATH = File.join(integration_path, "logstash")
+else
+  LOGSTASH_TEST_PATH = File.join(File.dirname(__FILE__), "..")
+end
+puts "[integration_spec] Running the test in #{LOGSTASH_TEST_PATH}"
+puts "[integration_spec] Running specs"
+
+RSpec.configure do |config|
+  config.order = "random"
```

Comment:
why not adding here the Logstash test helpers you wrote? so we are just including them as regular helpers?